ArchiSteamFarm/Core/ASF.cs

```diff
@@ -666,8 +688,7 @@ public static class ASF {
 		}
 
 		if (botName.Equals(SharedInfo.ASF, StringComparison.OrdinalIgnoreCase)) {
-			ArchiLogger.LogGenericInfo(Strings.GlobalConfigChanged);
-			await RestartOrExit().ConfigureAwait(false);
+			await OnConfigChanged().ConfigureAwait(false);
```

Comment:
maybe keep the log entry here? It will be easier to understand what caused an error when analyzing logs.

Reply:
Nah, we need more logic to load, validate and do stuff, this function already does more than it should.